config/pipelines/high_throughput_scrna_core_library_prep.wip.yml

```diff
@@ -0,0 +1,30 @@
+scRNA Core Chromium GEM-X 5p Aggregation:
+  pipeline_group: scRNA Core Library Prep
+  filters:
+    request_type_key: limber_scrna_core_chromium_gem_x_5p_aggregation
+  relationships:
+    # 2 input routes to this pipeline:
+    # LRC GEM-X 5p cDNA PCR XP coming through SeqOps
+    # LRC GEM-X 5p cDNA Input coming direct from faculty
+    LRC GEM-X 5p cDNA Input: LRC GEM-X 5p Aggregate
```

Comment:
Noticed we're still not consistent with the overall naming. Some names are 5p (or 5 prime) GEM-X (e.g. library types), some are GEM-X 5p (purposes). Probably should fix that one way or the other.

Also, did we decide for aggregation that the submission template would be generic, but we'd have specific request types and a range of different purpose names to prevent mixtures of plates being aggregated?
So we might have to add HT and 3p variations of these purposes?
e.g.

Reply:
No, in the SS PR it's a specific submission template and request type for these acceptable_purposes.

So we'd have to add further specific versions of all this for each variation they come up with.

Reply:
Looks like everything is 'GEM-X 5p' apart from the library type - I'll update that.

I didn't modify the aggregation submission as part of this but was planning to create a new story to review that and the earlier cDNA prep one, after our meeting with Liz and Abby. I think we probably do want to make the aggregation submission more generic. And we talked about taking the library type out of the cDNA prep one.

Reply:
Yep sounds good